src/main.rs

```diff
@@ -202,17 +203,26 @@ fn init_log() {
 #[cfg(not(feature = "logging"))]
 fn init_log() {}
 
-fn get_languages(env_lang: Option<&str>, env_language: Option<&str>) -> Vec<String> {
+fn get_languages(
```

Comment:
Maybe add a comment indicating the precedence?

```rust
/// Return language(s) with the following precedence:
///
/// 1. CLI language parameter (`-L`)
/// 2. Language as set in the config file
/// 3. If the `$LANG` env var is set: `$LANGUAGE` (list separated by `:`) followed by `$LANG` (single language)
/// 4. Fallback: English
```

Reply:
Added.